src/app/components/MediaLoader/index.styles.ts

```diff
@@ -20,6 +20,7 @@ export default {
     ({ spacings }: Theme) =>
       css({
         position: 'relative',
+        zIndex: '0',
```

Comment:
Do we the zindex CSS and the position relative now we have space between this and the player?